### ISSUE TITLE
Installation on windows using zserge instead of webview

### ISSUE THREAD

**Issue by SGarno** [guark/guark#31]
**Device:**
 - OS: Windows
 - Version: 10
 - Go Version: 1.15

**Describe the bug**
When running the installation on Windows, it is trying to use zserge/webview instead of webview/webview.

```
go get: github.com/zserge/webview@v0.0.0-20200516134046-4caf698fad53 updating to
        github.com/zserge/webview@v0.0.0-20200724072439-e0c01595b361: parsing go.mod:
        module declares its path as: github.com/webview/webview
                but was required as: github.com/zserge/webview
```

Cloning from the guark repository itself, seems to compile ok, but when trying to *init* I get an error:

```2020/11/03 07:43:39 Request error: 404 for https://api.github.com/repos/guark/-vue/contents/guark.yaml```

**To Reproduce**
1. go get -u github.com/guark/guark/cmd/guark

**Expected behavior**
Should use github.com/webview/webview dependency and not the older zserge/webview

**Comment by melbahja** [comment]
this issue will be resolved after releasing v0.1.0, probably tonight or tomorrow morning :)...

**Comment by SGarno** [comment]
Cool!   I have used webview when it was under zserge, but had to abandon because it was not compatible with some of the vue libraries I was using.   I am hoping that this new webview will work, and having your cli to build everything will be great!

**Comment by melbahja** [comment]
v0.1.0 released can you check it now ?

**Comment by SGarno** [comment]
Wow!   That was fast!

However, I am still getting the same error as when I did a clone on the repo.

```2020/11/03 11:08:41 Request error: 404 for https://api.github.com/repos/guark/-vue/contents/guark.yaml```

**Comment by melbahja** [comment]
try ‘guark init —template vue’

**Comment by SGarno** [comment]
Yeah, that was my bad.

Ok, some more fixes....

Looks like the template {{AppPkg}} was not expanded in `go.mod`, `app.go` and `config.go`.   I went ahead and did that manually.

Also, in your guart-build.yaml, it is required to change the path for the compilers.    I had to change mine to:

```
windows:
  cc: C:/TDM-GCC-64/bin/x86_64-w64-mingw32-gcc
  cxx: C:/TDM-GCC-64/bin//x86_64-w64-mingw32-g++
  ldflags: "-H windowsgui"
  windres: C:/TDM-GCC-64/bin/windres
```

Lastly, I am getting errors in the generated `embed.go`.    It is not escaping the paths properly:

```
⠇ Building Windows App...# example.com/test/myApp/lib
lib\embed.go:13:5: unknown escape
lib\embed.go:15:3: duplicate key "" in map literal
        previous key at lib\embed.go:13:3
lib\embed.go:15:5: unknown escape
lib\embed.go:16:3: duplicate key "" in map literal
        previous key at lib\embed.go:13:3
lib\embed.go:16:5: unknown escape
lib\embed.go:17:3: duplicate key "" in map literal
        previous key at lib\embed.go:13:3
lib\embed.go:17:5: unknown escape
lib\embed.go:18:3: duplicate key "" in map literal
        previous key at lib\embed.go:13:3
lib\embed.go:18:5: unknown escape
```

In the embed.go:13 is this line:

```       "\css\app.1748279e.css": ...```

It should be

```       "\\css\\app.1748279e.css": ...```

or, it might even work (even on windows) as 

```       "/css/app.1748279e.css": ...```

Is there anything I can change to fix this?

**Comment by melbahja** [comment]
I don’t have a windows machine to test this, but when you run guark init it should ask you to type your module name.

can you try with: 
guark init —template vue —mod github.com/yourname/yourapp

**Comment by melbahja** [comment]
embed files path should be with / because it will be served from a local http server

**Comment by SGarno** [comment]
Here is the exact command and errors:

```
D:\Development\guark>guark init --template vue --mod example.com/test/myapp
✔ Remote template validated
✔ Template downloaded successfully.
⏺ Change the app module name to yours:.com/test/myapp
✔ App module name: example.com/test/myapp
✔ App module name refactored
⏺ Setup commands (Verify them before confirm):
  - "yarn install" (dir: ui)
  - "go mod download"
  - "go mod tidy"
  - "go mod verify"
Do you want to run this commands on your machine (Enter for No): Y
Do you want to run this commands on your machine (Enter for No): Y
warning " > sass-loader@8.0.2" has unmet peer dependency "webpack@^4.36.0 || ^5.0.0".
⠧ Running setup command: go mod downloadgo: errors parsing go.mod:
D:\Development\guark\go.mod:1: usage: module module/path
exit status 1
```

Here is the go.mod file:

```
module {{AppPkg}}

go 1.15

require (
	github.com/guark/guark v0.1.0
	github.com/guark/plugins v0.0.0-20201102173241-c94a1e1f818e
)
```

You can see that `{{AppPkg}}` was not expanded.

After manually fixing the token, I ran a guark build:

```
D:\Development\guark>guark build
✔ Build Initialized   🔨
✔ Build App UI        🎨
✔ Build Embeded Files 🗃️
✔ Build Meta Files    ℹ️
⠼ Building Windows App...# runtime/cgo
exec: "/usr/bin/x86_64-w64-mingw32-gcc": file does not exist
⠏ Building Windows App...# example.com/test/myapp/lib
lib\embed.go:13:5: unknown escape
lib\embed.go:15:3: duplicate key "" in map literal
        previous key at lib\embed.go:13:3
lib\embed.go:15:5: unknown escape
lib\embed.go:16:3: duplicate key "" in map literal
        previous key at lib\embed.go:13:3
lib\embed.go:16:5: unknown escape
lib\embed.go:17:3: duplicate key "" in map literal
        previous key at lib\embed.go:13:3
lib\embed.go:17:5: unknown escape
lib\embed.go:18:3: duplicate key "" in map literal
        previous key at lib\embed.go:13:3
lib\embed.go:18:5: unknown escape
exit status 2
⠋ Building Windows App...
```

What do I need to change to make the generated embed.go use / instead of \?

**Comment by SGarno** [comment]
I got it.    I went into embed.go:56 and changed:

```
		embeds = append(embeds, &Item{
			ID:   strings.Replace(files[i], e.Root, "", 1),
			Path: files[i],
			Data: gzdata.Bytes(),
		})
```
to 
````
		embeds = append(embeds, &Item{
			ID:   strings.Replace(strings.Replace(files[i], e.Root, "", 1), "\\", "/", -1),
			Path: files[i],
			Data: gzdata.Bytes(),
		})
```

Bit of a hack, but now it compiles.

However, window is blank.   It does not start the ui.

How can I help track this down?

**Comment by melbahja** [comment]
Blank window you can fix it with:
```bash
CheckNetIsolation.exe LoopbackExempt -a -n="Microsoft.Win32WebViewHost_cw5n1h2txyewy"
```
because windows webview by default blocks localhost!

About the embed error, can you please open a new issue for it.

**Comment by SGarno** [comment]
Awesome!   Never would have thought that.   It is now working!     👍 👍

**Comment by melbahja** [comment]
Thanks, If you use `chrome`, I think this issue not exists if you use chrome as a engine.

**Comment by SGarno** [comment]
This is brilliant!    I actually have [Quasar](https://quasar.dev) now running in a go app.....this is exactly what I have been looking for!   And I didn't have to do anything special to babel or vue config to make it work.   Absolutely brilliant!

**Comment by melbahja** [comment]
This issue is fixed in `v0.1.1`.
